client/src/app/queries/assessments.ts

```diff
@@ -262,3 +264,33 @@ export const useFetchAssessmentsWithArchetypeApplications = () => {
     isLoading: assessmentsLoading || isArchetypesLoading,
   };
 };
+
+export const useFetchAllAssessmentsWithArchetypes = (
+  archetypes: Archetype[]
```

Comment:
Can just have `[]` as the default value and allow the param to be optional:
```suggestion
  archetypes: Archetype[] = []
```